_assets/css/_landing.scss

```diff
@@ -1197,52 +1122,49 @@ Stylesheet: Landing Page Stylesheet
 .news-slider {
   @include breakpoint(medium down) {
     padding: rem-calc(0 0 1);
-    margin-bottom: calc(#{5.6652083333333vw * 5} + #{rem-calc(96)}) !important;
+    margin-bottom: rem-calc(40) !important;
```

Comment:
Is the `!important` definitely required? Might be a sign that rules are over-specified elsewhere.